steps/common/steps.go

```diff
@@ -67,12 +69,20 @@ func (cs Steps) InitializeSteps(ctx context.Context, scenCtx *godog.ScenarioCont
 		return fmt.Errorf("mismatch of values: expected '%s', actual '%s'", e, v)
 	})
 	scenCtx.Step(`^I store my local ip in context "([^"]*)"$`, func(key string) error {
-		cmd := exec.Command("hostname", "-i")
-		stdoutStderr, err := cmd.CombinedOutput()
+		var ip string
+		netInterfaceAddresses, err := net.InterfaceAddrs()
 		if err != nil {
-			return fmt.Errorf("error executing `%s` command %v", cmd, string(stdoutStderr))
+			return fmt.Errorf("error getting local interfaces %w", err)
+		}
+		for _, netInterfaceAddress := range netInterfaceAddresses {
+			networkIP, ok := netInterfaceAddress.(*net.IPNet)
+			if ok && !networkIP.IP.IsLoopback() && networkIP.IP.To4() != nil {
```

Comment:
I will suggest to add another step:

`I store my local ip6 in context "([^"]*)`